### PR TITLE
Fix for duplicate serial reported by JMicron JMS561U

### PR DESCRIPTION
**Pull request by mmazerolle**
Fix for JMicron JMS561U USB3.0 SATA reporting duplicate serial #

Ref: https://github.com/openmediavault/openmediavault/pull/746

Tested on Raspberry Pi 4 with 2x JMS561U with 2x 3.5" 2TB HDD

### CODE REVIEW

File: deb/openmediavault/etc/udev/rules.d/99-openmediavault-dev-disk-by-id.rules [hunk 1]
```diff
@@ -33,6 +33,16 @@ ACTION=="add", KERNEL=="sd*", SUBSYSTEMS=="usb", \
   ENV{ID_SERIAL}="USB-%c-$env{ID_INSTANCE}", \
   ENV{ID_SERIAL_SHORT}="%c"
 
+# JMicron JMS561U two ports SATA 6Gb/s bridge
+ACTION=="add", KERNEL=="sd*", SUBSYSTEM=="block", \
+  ENV{ID_VENDOR_ID}=="152d", ENV{ID_MODEL_ID}=="1561", \
+  PROGRAM="/usr/lib/udev/serial_id %N", \
```
Comment (votdev) [inline deb/openmediavault/etc/udev/rules.d/99-openmediavault-dev-disk-by-id.rules:39]: Does it work when you use `PROGRAM="serial_id %N", \` here?

File: deb/openmediavault/etc/udev/rules.d/99-openmediavault-dev-disk-by-id.rules [hunk 2]
```diff
@@ -33,6 +33,16 @@ ACTION=="add", KERNEL=="sd*", SUBSYSTEMS=="usb", \
   ENV{ID_SERIAL}="USB-%c-$env{ID_INSTANCE}", \
   ENV{ID_SERIAL_SHORT}="%c"
 
+# JMicron JMS561U two ports SATA 6Gb/s bridge
```
Comment (votdev) [inline deb/openmediavault/etc/udev/rules.d/99-openmediavault-dev-disk-by-id.rules:36]: Do you have a link to the device?

**Review by votdev — CHANGES_REQUESTED** [review]

File: deb/openmediavault/etc/udev/rules.d/99-openmediavault-dev-disk-by-id.rules (hunk 2 above)
Comment (mmazerolle) [reply deb/openmediavault/etc/udev/rules.d/99-openmediavault-dev-disk-by-id.rules:36]: https://www.sabrent.com/product/EC-DFLT/usb-3-0-sata-external-hard-drive-lay-flat-docking-station-2-5-3-5in-hdd-ssd/

File: deb/openmediavault/etc/udev/rules.d/99-openmediavault-dev-disk-by-id.rules (hunk 1 above)
Comment (mmazerolle) [reply deb/openmediavault/etc/udev/rules.d/99-openmediavault-dev-disk-by-id.rules:39]: No, it does not. udevadm test /sys/block/sdX reports failure to execute serial_id without the explicit path to /usr/lib/udev/

Comment (votdev) [reply deb/openmediavault/etc/udev/rules.d/99-openmediavault-dev-disk-by-id.rules:39]: OK, good to know that. Thanks for testing this out.

File: deb/openmediavault/etc/udev/rules.d/99-openmediavault-dev-disk-by-id.rules (hunk 2 above)
Comment (votdev) [inline deb/openmediavault/etc/udev/rules.d/99-openmediavault-dev-disk-by-id.rules:36]:
```suggestion
# JMicron JMS561U two ports SATA 6Gb/s bridge
# https://www.sabrent.com/product/EC-DFLT/usb-3-0-sata-external-hard-drive-lay-flat-docking-station-2-5-3-5in-hdd-ssd/
```

**Review by votdev — CHANGES_REQUESTED** [review]

**Review by votdev — CHANGES_REQUESTED** [review]
Finally please make sure the PR fulfills the [contribution rules](https://openmediavault.readthedocs.io/en/latest/development/contribute.html#contribution-rules).

Squash the PR to a single commit and add the `Signed-off-by` line in the commit message.

**Comment by mmazerolle** [comment]
>> Squash the PR to a single commit and add the Signed-off-by line in the commit message.

Anything I need to do on my end? Do you have squash commits enabled on the repo?

**Comment by votdev** [comment]
> > > Squash the PR to a single commit and add the Signed-off-by line in the commit message.
> 
> Anything I need to do on my end? Do you have squash commits enabled on the repo?

Yes, squash the PR manually.

**Comment by votdev** [comment]
https://github.com/openmediavault/openmediavault/commit/3023d4ea362055f954c3e6c994a44ca13642470d superseds this PR because you didn't have squashed the 4 commits into one and the `Signed-off-by` line with your email address was still missing.

**Pull request closed without merging**